index.js

```diff
@@ -81,7 +81,9 @@ function choo () {
       if (model.subscriptions) {
         assert.ok(Array.isArray(model.subscriptions, 'subs must be an array'))
         model.subscriptions.forEach(function (sub) {
-          sub(send)
+          document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I wonder if swapping these two lines would work?
```javascript
document.addEventListener('DOMContentLoaded', function () {
  model.subscriptions.forEach(function (sub) { sub(send) }
}
```
That way you avoid creating an event listener for every subscription and hopefully achieve the same result.